### ISSUE TITLE
Recommended way to use nanopolish for diploid SNV calling for nanopore wgs consortium data?

### ISSUE THREAD

**Issue by pjedge** [jts/nanopolish#634]
Hi, I am running nanopolish for the purpose of benchmarking against our own diploid SNV calling method for nanopore reads. I am using the NA12878 rel6 data from the [nanopore wgs consortium](https://github.com/nanopore-wgs-consortium/NA12878/blob/master/Genome.md) and comparing against the GIAB variant set (v3.3.2) for hg38 chromosome 20. I just wanted to check that I am using the best possible methodology with this data set. The commands that I am using to run nanopolish (version 0.11.1) variants are of the form:

```
nanopolish variants --threads 4 --ploidy 2 --window chr20:1-1000000 --reads data/NA12878.hg38/aligned_reads/ont/ont.minimap2.all.30x.fastq.gz --bam data/NA12878.hg38/aligned_reads/ont/ont.minimap2.all.30x.bam --genome data/genomes/hg38.fa --outfile data/NA12878.hg38/variants/nanopolish.ont.minimap2.30x.unfiltered/20.1.1000000.vcf
```
I am processing the chromosome in chunks of 1 MB and recombining the vcf afterward. I am also filtering the variants for a maximum depth (TotalReads) of 67.

Compared to the ground truth GIAB variants I am getting a precision of 0.9346 and recall of 0.9009 for SNVs at QUAL=35.5. Is this in the ballpark that I should expect for human diploid data variant accuracy? In particular, I was wondering whether the best practice for this particular data set might involve accounting for methylation as described in [this blog post](http://simpsonlab.github.io/2017/10/31/methylation-aware/).

**Comment by jts** [comment]
Hi Peter,

Thanks for checking in about the best practices. To add methylation-aware calling use the `-q cpg` flag. Applying a strand bias filter improves nanopolish's calls, but we haven't integrated that directly into the software yet - its on our TODO list.

Jared

**Comment by jts** [comment]
There's now a branch with the strand bias test implemented:

https://github.com/jts/nanopolish/tree/fisher_strand_bias

Applying the test as a VCF FILTER is currently commented out [1], but a quick test on a small GIAB region improves precision from 0.95 to 0.98. I'll merge this after a bit more testing and when I select a good threshold

[1] https://github.com/jts/nanopolish/blob/fisher_strand_bias/src/common/nanopolish_variant.cpp#L441

**Comment by jts** [comment]
Merged into v0.12
